virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -10,344 +10,282 @@ ms.service: windows-containers
 
-In order to see which containers are connected to a specific network and the IPs associated with these container endpoints you can run the following.
+```powershell
+Get-Service vfpext
 
-```none
-C:\> docker network inspect <network name>
+# This should indicate the extension is Running: True 
+Get-VMSwitchExtension  -VMSwitchName <vSwitch Name> -Name "Microsoft Azure VFP Switch Extension"
 ```
+-- KALLIE TODO --
 
-### Specify the Name of a Network to the HNS Service
+## Tips & Insights
+Here's a list of handy tips and insights, inspired by common questions on Windows container networking that we hear from the community...
 
-**Ordinarily, when you create a container network using `docker network create`, the network name that you provide is used by the Docker service but not by the HNS service.**
-
-If you are creating a network, you can specify the name that it is given by the HNS service using the option, `-o com.docker.network.windowsshim.networkname=<network name>` to the `docker network create` command. For instance, you might use the following command to create a transparent network with a name that is specified to the HNS service:
+### Moby Linux VMs use DockerNAT switch with Docker for Windows (a product of [Docker CE](https://www.docker.com/community-edition)) instead of HNS internal vSwitch 
+Docker for Windows (the Windows driver for the Docker CE engine) on Windows 10 will use an Internal vSwitch named 'DockerNAT' to connect Moby Linux VMs to the container host. Developers using Moby Linux VMs on Windows should be aware that their hosts are using the DockerNAT vSwitch rather than the vSwitch that is created by the HNS service (which is the default switch used for Windows containers). 
 
+### To use DHCP for IP assignment on a virtual container host enable MACAddressSpoofing 
+If the container host is virtualized, and you wish to use DHCP for IP assignment, you must enable MACAddressSpoofing on the virtual machine's network adapter. Otherwise, the Hyper-V host will block network traffic from the containers in the VM with multiple MAC addresses. You can enable MACAddressSpoofing with this PowerShell command:
 ```none
-C:\> docker network create -d transparent -o com.docker.network.windowsshim.networkname=MyTransparentNetwork MyTransparentNetwork
+PS C:\> Get-VMNetworkAdapter -VMName ContainerHostVM | Set-VMNetworkAdapter -MacAddressSpoofing On
 ```
+### Creating multiple transparent networks on a single container host
+If you wish to create more than one transparent network you must specify to which (virtual) network adapter the external Hyper-V Virtual Switch should bind. To specify the interface for a network, use the following syntax:
 
-#### Example: Default HNS Naming Behavior
+--- @Jason: Is this syntax right?
```

Comment:
Yes, syntax is correct is users are relying on DHCP to obtain IP addresses for containers. If users want to perform static IP assignment for containers attached to a transparent network, then they also need to add --subnet and --gateway parameters.